docs/site/Project-layout-reference.md

```diff
@@ -0,0 +1,118 @@
+---
+
+LoopBack 4 project files and directories are in the _application root
+directory_. Within this directory the standard LoopBack project structure has
+these sub-directories:
+
+- `src` - Node application scripts and configuration files.
+- `public` - Client JavaScript, HTML, and CSS files.
+- `definitions` - API and product definition YAML files (**IBM API Connect
+  only**).
+
+{% include tip.html content="By LoopBack naming conventions, artifacts such as models are grouped under their sub-directories. Also, with LoopBack artifact generators, they create class names in CamelCase and file names are in snake-case. For example, if you create a model named `MyUserModel` with the model generator, it creates files `my-user.model.ts` under `src/models`. See [Naming Convention](Command-line-interface.md#naming-convention) for more information.
+" %}
+
+<table style="font-size: 90%;">
+  <thead>
+    <tr>
+      <th width="200">File or directory</th>
+      <th>Description</th>
+      <th width="180">How to access in code</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">Top-level application directory</th>
+    </tr>
+    <tr>
+      <td><code>/node-modules</code> directory</td>
+      <td>Contains Node packages as specified as dependencies in <code>package.json</code>. Update with <code>npm install</code>.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>
+          <code><a href="">package.json</a></code>
+      </td>
+      <td>
+        Standard npm package specification. Use it to set up package dependencies, among other things. This file must be in the application root directory. See <a href="https://docs.npmjs.com/files/package.json"><code>npm-package.json</code> for information.</a>.
+      </td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td><code>README.md</code></td>
+      <td>Stub file for internal documentation.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">/src directory - Node application files</th>
+    </tr>
+    <tr>
+      <td><code>/controllers</code> directory</td>
+      <td>A sub-directory for all controller files. See <a href="Controller.html">Controller</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>/datasources</code> directory</td>
+      <td>A sub-directory for all dataSource files. See <a href="DataSource.html">DataSource</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>/models</code> directory</td>
+      <td>A sub-directory for all model files. See <a href="Model.html">Model</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>/repositories</code> directory</td>
+      <td>A sub-directory for all repository files. See <a href="Repository.html">Repository</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>application.ts</code></td>
```

Comment:
Different from [LB3 docs](https://loopback.io/doc/en/lb3/Project-layout-reference.html), I don't have extra pages as examples for these files. Please let me know if you think they are needed.